install/tekton-dashboard-deployment.yaml

```diff
@@ -33,3 +33,16 @@ spec:
         env:
         - name: PORT
           value: "9097"
+        # Currently PipelineRuns MUST happen in the same namespace as the install namespace of the Tekton dashboard
+        # the current requirement is that the below namespace values match 
+        - name: PIPELINE_RUN_NAMESPACE
+          value: "replace_me"
+        - name: INSTALLED_NAMESPACE
+          value: "replace_me"
```

Comment:
Please remove the `PIPELINE_RUN_NAMESPACE` and `INSTALLED_NAMESPACE` environment variables. I can't find them being referenced in the code, so I'm not sure what they're supposed to add.

Also, the `createPipelineRun` function seems capable of creating a PipelineRun in any namespace, so why would we want to limit it to one `PIPELINE_RUN_NAMESPACE`?

Reply:
Good spot, looks like these aren't needed anymore, I have removed them